### PR TITLE
GG-7227 email-verification: update README with new screenshots and text

### PR DESCRIPTION
**Pull request by szabolcs-szilagyi-hmrc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,15 +239,25 @@ Replace example values with the appropriate values for your service to recommenc
 {
   "credId":"0000000026936462",
   "continueUrl":"/student-tax-checker/verify-email/journey-completed",
-  "origin":"stc",
+  "origin":"stc", // Will be displayed in the email as signature if labels.en.userFacingServiceName is null
   "deskproServiceName":"student-tax-checker",  // Optional, if absent then 'origin' will be used
   "accessibilityStatementUrl":"/accessibility-statement/stc",
-  "pageTitle": "Student Tax Checker", // Optional; will be displayed in the GOV.UK heading of the page
+  "pageTitle": "Student Tax Checker", // Optional, will be displayed in the email as sub title if labels.en.pageTitle is null
```
Comment (tim-holbrook) [inline README.md]: "is null" => "is missing"

File: README.md [hunk 2]
```diff
@@ -233,15 +239,25 @@ Replace example values with the appropriate values for your service to recommenc
 {
   "credId":"0000000026936462",
   "continueUrl":"/student-tax-checker/verify-email/journey-completed",
-  "origin":"stc",
+  "origin":"stc", // Will be displayed in the email as signature if labels.en.userFacingServiceName is null
```
Comment (tim-holbrook) [inline README.md]: "is null" => "is missing"

File: README.md (hunk 1 above)
Comment (tim-holbrook) [reply README.md]: The Lock Screen / English is showing the Welsh version (my mistake): 
https://github.com/hmrc/email-verification/blob/e9b04e43b52ce8758aa7589a1eb84d38b1d6d850/README.md#lock-screen---max-email-address-attempts-exceeded
Please revise /doc-images/Locked_MaxEmailAttempts.png file, thx.  I've sent you the corrected version.

File: conf/app.routes [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # microservice specific routes
 
-POST        /verification-requests                     uk.gov.hmrc.emailverification.controllers.EmailVerificationController.requestVerification
-POST        /verified-email-addresses                  uk.gov.hmrc.emailverification.controllers.EmailVerificationController.validateToken
-POST        /verified-email-check                      uk.gov.hmrc.emailverification.controllers.EmailVerificationController.verifiedEmail
+POST        /verification-requests                     uk.gov.hmrc.emailverification.controllers.EmailVerificationController.requestVerification()
```
Comment (tim-holbrook) [inline conf/app.routes:6]: Please can you place at the top:
POST /verify-email xxxxx
GET /verification-status/:credId xxxx
... at the top of the file.

Next, add a comment above all the others that says "DEPRECATED endpoints are below:"

**Review by tim-holbrook — DISMISSED** [review]

**Review by tim-holbrook — DISMISSED** [review]

**Comment by szabolcs-szilagyi-hmrc** [comment]
"The merge-base changed after approval." this error message stops us Approving the PR. The internet says we should close and reopen an ew PR which solves the problem.

**Review by tim-holbrook — APPROVED** [review]

**Review by tim-holbrook — APPROVED** [review]
Approved again, as Jira still has these as Open

**Pull request merged**